internal/discover/graphics.go

```diff
@@ -371,3 +374,15 @@ func (s selectDeviceByPath) MountIsSelected(Mount) bool {
 func (s selectDeviceByPath) HookIsSelected(Hook) bool {
 	return true
 }
+
+// xdgDataDirs finds the paths as specified in the environment variable XDG_DATA_DIRS.
+// See https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html.
+func xdgDataDirs() []string {
+	dirs, exists := os.LookupEnv("XDG_DATA_DIRS")
+
+	if !exists {
```

Comment:
Should this be:
```suggestion
	if !exists || dirs == "" {
```
given:
> If $XDG_DATA_DIRS is either not set or empty, a value equal to /usr/local/share/:/usr/share/ should be used.

Reply:
yes, definitely!